src/NubankClientDemo/Program.cs

```diff
@@ -18,32 +19,37 @@ static async Task Main()
             var password = Console.ReadLine().Trim();
             var nubankClient = new Nubank(login, password);
             var result = await nubankClient.LoginAsync();
+
             if (result.NeedsDeviceAuthorization)
             {
                 Console.WriteLine("You must authenticate with your phone to be able to access your data.");
                 Console.WriteLine("Scan the QRCode below with you Nubank application on the following menu:");
                 Console.WriteLine("Nu(Seu Nome) > Perfil > Acesso pelo site");
                 Console.WriteLine();
-                var directory = Path.Combine(Directory.GetCurrentDirectory(), "qrcodes");
-                if (!Directory.Exists(directory))
-                {
-                    Directory.CreateDirectory(directory);
-                }
-
-                var filename = Path.Combine(directory, DateTime.Now.ToString("yyyyMMddHH:mm") + ".jpg");
-                result
-                    .GetQrCodeAsBitmap()
-                    .Save(filename);
-
-                Console.WriteLine($"Open the file '{filename}' and use your phone to scan and after this press any key to continue...");
+                Console.WriteLine(result.GetQrCodeAsAscii());
```

Comment:
@natenho Didn't you have problem authorizing this qrcode printed in the console with your smarpthone? I changed this to generate a file after a problem that I had, the app didn't recognize this as a valid qrcode.

Reply:
It may depend on the distance of the camera from screen, but the QR code in the console is fine! BTW, I would suggest you to not depend directly on QR library, I'll open an issue to detail the idea.

Reply:
@natenho ok,  I will take a look as soon as possible.